### PR TITLE
Replace bbl.bin with fw_payload.bin per #234.

### PR DESCRIPTION
**Pull request by zchn**
I actually haven't got it working so this is just an update purely based on the discussions in https://groups.google.com/g/keystone-enclave-forum/c/4XGOQ0khzEI?pli=1

### CODE REVIEW

File: docs/source/Getting-Started/Running-Keystone-on-Hardware.rst [hunk 1]
```diff
@@ -62,10 +62,9 @@ or example script below).
 Load Linux Image
 ################
 
-The hifive build process generates a bbl.bin in
-``<build directory>/bbl.bin``. Flash this to the Linux partition on the
-card. (Note that this is a relocated version of the bbl binary used
-for QEMU)
+Flash the ``fw_payload.bin`` generated by the keystone build process above
```
Comment (zchn) [inline docs/source/Getting-Started/Running-Keystone-on-Hardware.rst:65]: @dayeol is this accurate? How is this different from the "Building Keystone" section above?

File: docs/source/Getting-Started/Running-Keystone-on-Hardware.rst [hunk 2]
```diff
@@ -33,11 +33,11 @@ SiFive-compatible Linux and SM.
 This includes some patches for DTB compatibility.
 Also, the build will forcibly use initramfs for a simpler deployment.
 
-Once you have built the image, you will see ``riscv-pk.build/bbl`` under your
-build directory.
+Once you have built the image, you will see 
+``sm.build/platform/generic/firmware/fw_payload.bin`` under your build directory.
```
Comment (dayeol) [inline docs/source/Getting-Started/Running-Keystone-on-Hardware.rst:37]: This is not true, `make image` will not generate fw_payload.bin under your build directory.

File: docs/source/Getting-Started/Running-Keystone-on-Hardware.rst [hunk 3]
```diff
@@ -33,11 +33,11 @@ SiFive-compatible Linux and SM.
...
 
-Separately, ``make image`` will also generate ``bbl.bin`` under your build directory.
-This is the file that you want to reflash the SD card with.
+Separately, ``make image`` will also generate ``fw_payload.bin`` under your build
```
Comment (dayeol) [inline docs/source/Getting-Started/Running-Keystone-on-Hardware.rst:39]: This is not true, make image will not generate fw_payload.bin under your build directory.

File: docs/source/Getting-Started/Running-Keystone-on-Hardware.rst (hunk 1 above)
Comment (dayeol) [reply docs/source/Getting-Started/Running-Keystone-on-Hardware.rst:65]: Yes this is accruate. flash `fw_payload.bin` instead of `bbl.bin`

**Review by dayeol — CHANGES_REQUESTED** [review]

File: docs/source/Getting-Started/Running-Keystone-on-Hardware.rst (hunk 3 above)
Comment (zchn) [reply docs/source/Getting-Started/Running-Keystone-on-Hardware.rst]: Ack. Removed mentioning of `make image` as it is not needed. I can see the fw_payload.bin after just a `make`.

**Review by dayeol — APPROVED** [review]

**Pull request merged**